src/lstm_tsp.py

```diff
@@ -0,0 +1,32 @@
+"""
+lstm_tsp
+"""
+import torch
+
+from .model import BenchmarkLSTM
+from .time_series_predictor import TimeSeriesPredictor
+
+class LSTMTimeSeriesPredictor(TimeSeriesPredictor):
```

Comment:
I don't understand the need for this class

Reply:
The purpose was to isolate concerns and make a time series predictor resembling [sklearn fit and predict architecture](https://scikit-learn.org/stable/tutorial/statistical_inference/supervised_learning.html). Pros and cons here.